.changeset/neat-lies-know.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-catalog-react': minor
+---
+
+Cleanup and small fixes for the kind selector
```

Comment:
If there are no publicly facing changes, but rather just some bug fixes, then it might be better as a `patch` level change than `minor`?